CODEOWNERS

```diff
@@ -139,6 +139,7 @@
 /dts/bindings/can/                        @alexanderwachter
 /dts/bindings/*/nordic*                   @anangl
 /dts/bindings/*/nxp*                      @MaureenHelm
+/dts/bindings/sram/sifive,dtim0.yaml      @mateusz-holenko @kgugala @pgielda @nategraff-sifive
```

Comment:
Make this more generic like the two previous entries. Something like:
```
/dts/bindings/*/sifive*      @mateusz-holenko @kgugala @pgielda @nategraff-sifive
```

Reply:
Done